### PR TITLE
Add (artist-)card element with picture and subtitle

### PR DESCRIPTION
**Pull request by fgottesmann**
We created card components and add styling + responsiveness. Also fixed a bug which causes upstream errors. 
We were solving Issue #25

### CODE REVIEW

**Review by MaxMuecket — COMMENTED** [review]
lgtm!
maybe you could add the issue #25 in your comment.

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "serve": "vite preview",
     "prepare": "husky install",
     "test": "tsc && prettier --check . && eslint . && stylelint \"**/*.css\"",
-    "storybook": "start-storybook -p 6006",
+    "storybook": "start-storybook -s ./src -p 6006",
```
Comment (infoucarsinan) [inline package.json]: A new branch for that.

Comment (phlgr) [reply package.json]: This is needed for the PR to work.

**Review by phlgr — APPROVED** [review]
GJ

**Review by tmh9x — APPROVED** [review]

File: package-lock.json [hunk 2]
```diff
@@ -22435,7 +22435,6 @@
       "integrity": "sha512-qOtwgiqI3LMqT0eXYNV6ykp7qSu0LQGeXxy3wOBGuDDqAizfgnAjomYEWGFcyKp5ahV7HCRCjxbixAklFPUmyw==",
       "dev": true,
       "requires": {
-        "@babel/core": "^7.12.10",
```
Comment (lmachens) [inline package-lock.json]: This change is not related. Can you remove it from changes?

File: package.json (hunk 1 above)
Comment (lmachens) [reply package.json]: Can you describe this change? The path is different from this pull request:
https://github.com/lmachens/photoplay/pull/20/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R13

File: src/app/components/Card/Card.module.css [hunk 3]
```diff
@@ -0,0 +1,20 @@
+.artistCard {
```
Comment (lmachens) [inline src/app/components/Card/Card.module.css]: I would love to see BEM convention applied here :)

File: src/app/components/Card/Card.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+type CardProps = {
+  img: string;
+  artist: string;
+};
+
+function Card({ img, artist }: CardProps): JSX.Element {
+  return (
+    <div className={classes.artistCard}>
+      <img className={classes.artistImage} src={img} alt="" />
```
Comment (lmachens) [inline src/app/components/Card/Card.tsx]: What do you think about renaming `img` to `imgSrc`? Otherwise  `img` is an HTML element and a string.

File: src/app/components/Card/Card.tsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import classes from './Card.module.css';
+
+type CardProps = {
+  img: string;
+  artist: string;
+};
+
+function Card({ img, artist }: CardProps): JSX.Element {
```
Comment (lmachens) [inline src/app/components/Card/Card.tsx]: In the future, we will have more cards for episodes and movies. Can you rename the component to `ArtistCard`?

**Review by lmachens — COMMENTED** [review]
good job :)
Please update the pull request title

File: package.json (hunk 1 above)
Comment (fgottesmann) [reply package.json]: We made an assets folder and set our package.JSON to: 

   ` "storybook": "start-storybook -s src/app/assets -p 6006",`

File: src/app/components/Card/Card.module.css (hunk 3 above)
Comment (fgottesmann) [reply src/app/components/Card/Card.module.css]: BEM has been applied. Thanks for advice!

File: src/app/components/Card/Card.tsx (hunk 4 above)
Comment (fgottesmann) [reply src/app/components/Card/Card.tsx]: You´re absolutely right. We have made those changes

File: src/app/components/Card/Card.tsx (hunk 5 above)
Comment (fgottesmann) [reply src/app/components/Card/Card.tsx]: We renamed the components and also the files.

File: package-lock.json (hunk 2 above)
Comment (fgottesmann) [reply package-lock.json]: We have been reverting those changes to:
        `"@babel/core": "^7.12.10",`

**Review by lmachens — COMMENTED** [review]
Good job! Please rename folder `Card` to `ArtistCard`

**Review by lmachens — APPROVED** [review]
╰(*°▽°*)╯
